src/Features/Core/Portable/PullMemberUp/MembersPuller.cs

```diff
@@ -337,7 +337,7 @@ private static ISymbol GetSymbolsToPullUp(MemberAnalysisResult analysisResult)
                 node.GetCurrentNode(newDestination),
                 sourceImports,
                 generator,
-                options.PlaceSystemNamespaceFirst,
```

Comment:
This is the fix for anything using `MembersPuller`: Nothing ever wrote to this property.